packages/now-cli/test/integration.js

```diff
@@ -493,6 +493,35 @@ test('list the payment methods', async t => {
   t.true(stdout.startsWith(`> 0 cards found under ${contextName}`));
 });
 
+test('domains inspect', async t => {
+  const domainName = `inspect-${contextName}.org`;
```

Comment:
How did you choose this domain? 

I'm kinda worried about using .org because the [price cap was removed](https://arstechnica.com/tech-policy/2019/07/icann-eliminates-org-domain-price-caps-despite-lopsided-opposition/).

Is the test purchasing a new domain each time?

Reply:
Oh I see the `add` is for an existing domain. I see the other command is `now domains buy` 😅